examples/README.md

```diff
@@ -10,8 +10,8 @@ within the Flow framework on a variety of traffic problems. These examples are
 python files that may be executed either from terminal or via a text editor. 
 For example, in order to execute the non-RL Ring example we run:
 
-```shell
-python simulate.py ring
+```shell script
+python simulate.py "ring"
```

Comment:
does it actually need string quotes?

Reply:
it doesn't, but better practice for cases where the string might contain special characters (e.g. spaces)